package.json

```
@@ -1,6 +1,6 @@
 {
     "dependencies": {
-        "bootstrap-italia": "^1.2.4",
+        "bootstrap-italia": "^1.2.5",
```

Should we switch to the new version?
```suggestion
        "bootstrap-italia": "^1.2.6",
```